### ISSUE TITLE
Windows handling

### ISSUE THREAD

**Issue by alchem1ster** [vemel/handsdown#15]
**Describe the bug**
Wrong OS link symbol handler and path prediction on Windows

**To Reproduce**
1. Run `handsdown --external https://github.com/alchem1ster/<my_repository_name>/` on Windows
2. Get an error and wrong relative links in Docs

**PoC 1 - wrong path prediction**
```handsdown --external https://github.com/alchem1ster/<my_repository_name>/
INFO     Creating folder O:\Projects\<my_repository_name>\docs
INFO     Creating O:\Projects\<my_repository_name>\docs\_config.yml file
Traceback (most recent call last):
  File "O:\Python38\lib\runpy.py", line 194, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "O:\Python38\lib\runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "O:\Python38\Scripts\handsdown.exe\__main__.py", line 7, in <module>
  File "O:\Python38\lib\site-packages\handsdown\main.py", line 73, in main
    create_external_configs(args)
  File "O:\Python38\lib\site-packages\handsdown\main.py", line 28, in create_external_configs
    render_asset(
  File "O:\Python38\lib\site-packages\handsdown\utils\__init__.py", line 60, in render_asset
    content = (Path(ASSETS_PATH) / name).read_text()
  File "O:\Python38\lib\pathlib.py", line 1236, in read_text
    with self.open(mode='r', encoding=encoding, errors=errors) as f:
  File "O:\Python38\lib\pathlib.py", line 1222, in open
    return io.open(self, mode, buffering, encoding, errors, newline,
  File "O:\Python38\lib\pathlib.py", line 1078, in _opener
    return self._accessor.open(self, flags, mode)
FileNotFoundError: [Errno 2] No such file or directory: 'O:\\Python38\\lib\\site-packages\\handsdown\\settings.py\\assets\\gh_pages_config.yml'
```

**PoC 2 - wrong path symbol (will be 404 Error on GitHub)**
```
- [Dulwich](dulwich\index.md#dulwich)
    - [Archive](dulwich\archive.md#archive)
    - [Client](dulwich\client.md#client)
    - [Clone](dulwich\clone.md#clone)
    - [Config](dulwich\config.md#config)
    - [Diff Tree](dulwich\diff_tree.md#diff-tree)
    - [Errors](dulwich\errors.md#errors)
```

**Expected behavior**
No errors

**Fast and dirty workarounds for me**
1. https://github.com/vemel/handsdown/pull/14
2. `link = f"{link_path}{link}".replace("\\", "/")`  in [md_document.py#L235](https://github.com/vemel/handsdown/blob/cdbb39fc09cdb1852f73798e8a5aba6d91cb9ed7/handsdown/md_document.py#L235)

**Desktop**
- OS: Windows 10 Professional 21H2
- Python version: 3.8.10
- Handsdown version: 0.4.2 (installed via pip)

**Comment by alchem1ster** [comment]
And one more nuance - the standard GitHub branch is now `main` instead of `master`.

**Comment by vemel** [comment]
Sorry, it took so long. I have just released `0.5.0` with both fixes. THank you for the report and let me know if it works as it should now.

**Comment by alchem1ster** [comment]
> with both fixes

Но их было три 😄 
```
Traceback (most recent call last):
  File "O:\Python38\lib\runpy.py", line 194, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "O:\Python38\lib\runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "O:\Python38\Scripts\handsdown.exe\__main__.py", line 7, in <module>
  File "O:\Python38\lib\site-packages\handsdown\main.py", line 73, in main
    create_external_configs(args)
  File "O:\Python38\lib\site-packages\handsdown\main.py", line 28, in create_external_configs
    render_asset(
  File "O:\Python38\lib\site-packages\handsdown\utils\__init__.py", line 60, in render_asset
    content = (Path(ASSETS_PATH) / name).read_text()
  File "O:\Python38\lib\pathlib.py", line 1236, in read_text
    with self.open(mode='r', encoding=encoding, errors=errors) as f:
  File "O:\Python38\lib\pathlib.py", line 1222, in open
    return io.open(self, mode, buffering, encoding, errors, newline,
  File "O:\Python38\lib\pathlib.py", line 1078, in _opener
    return self._accessor.open(self, flags, mode)
FileNotFoundError: [Errno 2] No such file or directory: 'O:\\Python38\\lib\\site-packages\\handsdown\\settings.py\\assets\\gh_pages_config.yml'
```

**Comment by vemel** [comment]
This is a different issue, completely overlooked it! SHould be fixed in `0.5.1`

**Comment by alchem1ster** [comment]
All seems to be fixed 👍
